parlai/utils/misc.py

```diff
@@ -13,6 +13,7 @@
 import time
 from typing import Union, Optional, Set, Any, Dict, List
 import warnings
+from parlai.core.opt import Opt
```

Comment:
ahh yes I am aware this broke previous models, we should not add it back here however, those models should be re-saved if necessary (I have already done this with my old models)

Reply:
Yes, this is resolved in master already